camayoc/tests/qcs/api/v1/scans/test_network_scans.py

```diff
@@ -112,7 +63,7 @@ def prep_network_scan(source, cleanup, client):
     """
     cfg = config.get_config()
     auth_ids = []
-    for auth in cfg['qcs'].get('auths'):
+    for auth in cfg.get('auths'):
```

Comment:
I think you could access `cfg['auths']` here if the auths section is not defined it will raise an exception:

```
>>> for i in None:
...   print 'None'
... 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'NoneType' object is not iterable
```

So I think is better to raise a KeyError here. This will tell us `config object does not have the auths "section"`.